tidb-cloud/tidb-cloud-encrypt-cmek.md

```diff
@@ -0,0 +1,124 @@
+                "Condition": {
+                    "StringEquals": {
+                        "kms:CallerAccount": "<pingcap-account>",
+                        "kms:ViaService": "ec2.us-west-2.amazonaws.com"
+                    }
+                }
+            },
+            // S3-related policy
+            {
+                "Sid": "Allow TiDB cloud role to use KMS to store encrypted backup to S3",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::<pingcap-account>:root"
+                },
+                "Action": [
+                    "kms:Decrypt",
+                    "kms:GenerateDataKey"
+                ],
+                "Resource": "*"
+            },
+            ... // user's own admin access to KMS
+        ]
+    }
+    ```
+
+> **Note:**
+>
+> - `<pingcap-account>` is the account where your clusters run in. If you do not already know the account, you can [contact the technical support staff](/tidb-cloud/tidb-cloud-support.md).
+> - For EBS-related policy in the first block, refer to [AWS documentation](https://docs.aws.amazon.com/kms/latest/developerguide/conditions-kms.html#conditions-kms-caller-account).
+> - For S3-related policy in the second block, refer to [AWS blog](https://repost.aws/knowledge-center/s3-bucket-access-default-encryption).
+
+### Step 2. Create a new project and enable CMEK
+
+> **Note:**
+>
+> Currently the configuration is not available on the TiDB Cloud console. You can only configure the project using TiDB Cloud API.
+
+1. Create a new project and enable AWS CMEK using the [Create a project](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Project/operation/CreateProject) endpoint of TiDB Cloud API.
+
+    Make sure the `aws_cmek_enabled` field is set to `true`.
+
+2. Configure the KMS Key ARN for the specified region (e.g., us-east-1) under this project using the [Configure AWS CMEK](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/CreateAwsCmek) endpoint of TiDB Cloud API.
+
+### Step 3. Create a cluster
+
+Create a Dedicated Tier cluster hosted on AWS under the project created in Step 1. You need to ensure that the region where the cluster is located is the same as that in Step 2.
+
+> **Note:**
+>
+> When CMEK is enabled, the EBS volumes used by the nodes of the cluster and the S3 used for the cluster backups are encrypted using CMEK.
+
+## Rotate your CMEK
+
+You can configure [automatic CMEK rotation](http://docs.aws.amazon.com/kms/latest/developerguide/rotate-keys.html) on AWS KMS. With this rotation enabled, you do not need to update the Encryption at Rest project settings in TiDB Cloud, including the CMEK ID.
+
+## Revoke and restore CMEK
+
+If you need to temporarily revoke TiDB Cloud's access to CMEK, follow these steps:
+
+1. On the AWS KMS console, revoke the corresponding permissions and update the KMS Key policy.
+2. On the TiDB Cloud console, pause all the clusters in the project.
```

Comment:
Is this a must step? What if I just want to disable CMEK but keep the clusters running?

Reply:
This is a necessary step, please see the note information.
"After you revoke CMEK on AWS KMS, your running clusters are not affected"